src/single-spa-svelte.js

```diff
@@ -1,3 +1,5 @@
+import { mount as originalMount, unmount as originalUnmount } from "svelte";
```

Comment:
svelte is in devDependencies, which means that this import would fail in a library that installs single-spa-svelte. Traditionally, single-spa adapter libraries for frameworks use dependency injection of the framework  (pass in React/Angular/Vue libs via opts) rather than imports, due to the complexities of bundling the adapter libraries and/or using them as runtime shared dependencies in import maps.

Please switch to the mount and unmount functions being passed in as opts